src/customer-service/Actions/MainMenu.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+namespace customer_service
```

Comment:
@TeamCharles/team   We have some of these in `customer_service.Actions` and some in `customer_service`.  I think we should decide what we want and make the consistent.  I would suggest we move them all to `Actions`.